src/LibraryManager/Providers/Cdnjs/CdnjsCatalog.cs

```diff
@@ -194,7 +194,10 @@ public async Task<string> GetLatestVersion(string libraryName, bool includePreRe
             }
 
             string first = includePreReleases
-                ? (await GetLibraryVersionsAsync(group.DisplayName, cancellationToken).ConfigureAwait(false)).First()
+                ? (await GetLibraryVersionsAsync(group.DisplayName, cancellationToken).ConfigureAwait(false))
+                                                                                      .Select(v => SemanticVersion.Parse(v))
+                                                                                      .Max()
```

Comment:
Since this wasn't protected before, I assume there's no concern about the results being empty? (I think Max on an empty set throws).

Reply:
Since Cdnjs is a curated catalog, I admit we're making an assumption here that they'll always have at least one version in the catalog.  But I think that's a safe assumption.